ql/src/test/results/clientpositive/llap/udaf_number_format.q.out

```diff
@@ -92,4 +92,4 @@ FROM src
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@src
 #### A masked pattern was here ####
-0.0	NULL	NULL	NULL
+NULL	NULL	NULL	NULL
```

Comment:
We may remove this test case as `cbo_aggregate_reduce_functions_rule.q` covers the case.
https://github.com/apache/hive/blob/master/ql/src/test/queries/clientpositive/udaf_number_format.q